cmd/swarmctl/cluster/update.go

```diff
@@ -112,4 +135,6 @@ func init() {
 	updateCmd.Flags().String("log-driver", "", "Set default log driver for cluster")
 	updateCmd.Flags().StringSlice("log-opt", nil, "Set options for default log driver")
 	updateCmd.Flags().String("rotate-join-token", "", "Rotate join token for worker or manager")
+	updateCmd.Flags().Bool("rotate-unlock-key", false, "Rotate manager unlock key")
+	updateCmd.Flags().String("autolock", "", "Enable or disable manager autolocking (requiring an unlock key to start a stopped manager) by setting this to 'on' or 'off'")
```

Comment:
Hmm... and just check if it's changed before submitting it, you mean?  Does it count as changed even if you pass `--autolock false`?